Documentation/Contribute/ContributeWithGit.rst

```diff
@@ -66,6 +66,11 @@ binary tree (*/path/to/ITKSphinxExamples-build/Utilities*)::
   cd /path/to/ITKSphinxExamples-build/Utilities
   python ./CreateNewExample.py
 
+The script will ask for an *example_name* and a *class_name*.  The example_name
+might be *CreateANewExample* for an example that answers the question "How do I
```

Comment:
Too meta. Maybe `CreateANewExample` -> `ComputeTumorIntensityVariation`, and `itk::Statistics::MaskedImageToHistogramFilter` for a class in focus.

Reply:
Thank you.  Done.